packages/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.test.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import TypeAheadSelect from './TypeAheadSelect';
+import { TypeAheadSelect } from './TypeAheadSelect';
 
 test('TypeAheadSelect is working !!', () => {
   const component = shallow(
-    <TypeAheadSelect
-      clearButton
-      multiple
-      allowNew
-      options={['One', 'Two', 'Three']}
-    />
+    <p>
```

Comment:
should not need these p tags now that the import is good.

Reply:
The `<p>` tag is there because `shallow` does not provide DOM node reference for the immediate component so the `<p>` serves as a wrapper. Without it the tests are failing somewhere in `react-onclickoutside` -- previously this wasn't visible because the component was already wrapped.

Reply:
@dmiller9911 If there's a better solution for this in the tests please let me know. Googling doesn't seem to turn out anything useful.

Reply:
Ah makes sense.  Looking through their code there is a decent amount of DOM stuff going on.